payments-core/src/main/java/com/stripe/android/view/CardWidgetViewModel.kt

```diff
@@ -24,7 +25,7 @@ import kotlinx.coroutines.launch
 import kotlin.time.Duration.Companion.seconds
 
 internal class CardWidgetViewModel(
-    private val cbcEnabled: CbcEnabledProvider,
+    private val cbcEligible: () -> Boolean = { DEBUG },
```

Comment:
I’ve been a bit unclear here before, but the provider function should determine whether the widget is _eligible_ for CBC. Whether or not the feature is enabled is coming from `FeatureFlags.cardBrandChoice` now.